pkg/cloud/cloud.go

```diff
@@ -49,6 +49,9 @@ var (
 
 	// ErrAlreadyExists is returned when a resource is already existent.
 	ErrAlreadyExists = errors.New("resource already exists")
+
+	// ErrIdempotent is returned when another request with same idempotent token is in-flight.
+	ErrIdempotentParameterMismatch = errors.New("parameters on this idempotent request are inconsistent with parameters used in previous request(s)")
```

Comment:
where is this error used? are we returning this code anywhere in the driver?

Reply:
Error is used here: https://github.com/ppc64le-cloud/powervs-csi-driver/pull/39/files#diff-4e4f8ad2f057d794bd5842ed50e358451e6e1b3b20e5f2836289665bdfbf5625R124

Reply:
fixed